src/runtime/ModuleStore.js

```diff
@@ -152,9 +152,12 @@
               evaled.push(frame);
             }
           });
-          ex.stack = evaled.join('\n');
+          // Allocate new object, some platforms make ex.stack read-only.
+          ex = {
```

Comment:
We want to 'lose' the exception object because its contents reflects the ES5 transcoded underworld. The `stack` property contains some lines of light -- es6 frames -- and probably many lines of dark -- module loader and compiler frames.  

Arguably we ought to create a new `Error` object but the non-configurable `stack` makes that fail. Second best is a new object with the same properties so downstream code is effective.  Mutating the given Error object -- a habit from olden days -- is probably not the clearest solution. We are lying about the stack on purpose but at least we should be honest and let downstream code know we are lying if they ask.